### DISCUSSION TITLE
Error in FragPipe Analyst

### DISCUSSION THREAD

**Discussion by TeckYewLow** [Nesvilab/FragPipe-Analyst-doc#36] [General]
Hi,

I was loading the LFQ file and annotation. Suddenly I got this error message: "The labels/'run names' in the experimental design DID NOT match with lfq column names in maxquants proteinGroups file Run LFQ-Analyst with correct labels in the experimental design" 

Please advise me what I should do. For your information I chose "MaxLFQ" as intensity type. If I had chose "Intensity" as intensity type, it would run just fine.

**Comment by hsiaoyi0504** [comment]
@TeckYewLow Do you have MaxLFQ columns available in your combined_protein.txt file? I couldn't reproduce using our testing datasets. Or do you mind sharing your files to figure out what's the issue? yihsiao@umich.edu

**Comment by hsiaoyi0504** [comment]
Closed due to no response.
